include/PCH.h

```diff
@@ -31,6 +31,10 @@ void* operator new[](size_t size, size_t alignment, size_t alignmentOffset, cons
 #define WIN32_LEAN_AND_MEAN
 #include <Windows.h>
 
+#ifndef TRACY_SUPPORT
+	#undef TRACY_ENABLE
```

Comment:
Why undef? Where is the def?

Reply:
Hardcoded here :'(
<img width="404" alt="image" src="https://github.com/user-attachments/assets/6de2b737-d503-4ae7-8438-11a3637af1a5">